### PR TITLE
fixed add pageable users, products, and inventory

### PR DESCRIPTION
**Pull request by CVelazquezURJC**
;)

### CODE REVIEW

**Comment by JorgeGalingP** [comment]
# NICE

**Comment by JorgeGalingP** [comment]
# WORK

**Comment by JorgeGalingP** [comment]
# 😙

File: src/main/java/daw/spring/restcontroller/AdminDashboardRestController.java [hunk 1]
```diff
@@ -21,15 +26,21 @@
     private final NotificationService notificationService;
     private final DeviceService deviceService;
     private final HomeService homeService;
+    private final OrderRequestRepository orderRequestRepository;
+    private final UserRepository userRepository;
+    private final ProductRepository productRepository;
 
     @Autowired
-    public AdminDashboardRestController(UserService userService, ProductService productService, OrderRequestService orderRequestService, NotificationService notificationService, DeviceService deviceService , HomeService homeService) {
+    public AdminDashboardRestController(UserService userService, ProductService productService, OrderRequestService orderRequestService, NotificationService notificationService, DeviceService deviceService, HomeService homeService, OrderRequestRepository orderRequestRepository, UserRepository userRepository, ProductRepository productRepository) {
```
Comment (Hgmoa) [inline src/main/java/daw/spring/restcontroller/AdminDashboardRestController.java]: MAJOR Constructor has 9 parameters, which is greater than 7 authorized. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00107)

File: src/main/java/daw/spring/restcontroller/AdminDashboardRestController.java [hunk 2]
```diff
@@ -145,18 +158,19 @@ public ResponseEntity getUser(Principal principal, @PathVariable long id) {
...
-            List<OrderRequest> listOrdersNotcomplete = orderRequestService.findNotCompletedOrdersAll();
-            List<OrderRequest> listOrdersAreComplete = orderRequestService.findCompletedOrdersAll();
-            Map<String, List<OrderRequest>> ordersOut = new HashMap<>();
+            Page<OrderRequest> listOrdersNotcomplete = orderRequestRepository.findAllByCompletedIsFalse(page);
+            Page<OrderRequest> listOrdersAreComplete = orderRequestRepository.findAllByCompletedIsFalse(page);
+            Map<String, Page<OrderRequest>> ordersOut = new HashMap<>();
             ordersOut.put("OrdersNotComplete", listOrdersNotcomplete);
             ordersOut.put("OrdersComplete", listOrdersAreComplete);
-            return ResponseEntity.ok(ordersOut) ;
+            //return ResponseEntity.ok(ordersOut) ;
```
Comment (Hgmoa) [inline src/main/java/daw/spring/restcontroller/AdminDashboardRestController.java]: MAJOR This block of commented-out lines of code should be removed. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)

**Comment by Hgmoa** [comment]
SonarQube analysis reported 2 issues
* ![MAJOR][MAJOR] 2 major

Watch the comments in this conversation to review them.

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

**Pull request merged**